packages/cadl-ranch-specs/http/azure/arm/models/common-types/managed-identity/main.tsp

```diff
@@ -0,0 +1,37 @@
+import "@azure-tools/cadl-ranch-expect";
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-client-generator-core";
+import "./identity.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+@armProviderNamespace
+@service
+@versioned(Versions)
+@doc("Arm Common Types Managed Identity Provider management API.")
+namespace Azure.Arm.Models.CommonTypes.ManagedIdentity;
```

Comment:
namespace should be ResourceManager instead of Arm?

Reply:
The @armProviderNamespace decorator must be used to define the ARM namespace of the service.  This is best applied to the file-level namespace.

Reply:
The namespace has been changed to `Azure.ResourceManager.Models.CommonTypes.ManagedIdentity`.